### PR TITLE
[FIX] #25 path initial param issue

### PR DESCRIPTION
**Pull request by josepot**
Addresses #25

### CODE REVIEW

**Comment by josepot** [comment]
I should add some tests to at least "document" this behavior. I will do that tomorrow. I won't merge it until I don't have a few tests...

File: src/path.js [hunk 1]
```diff
@@ -26,3 +29,7 @@ export default isInner => {
     return arguments.length === 1 ? fn : fn(arguments[1])
   }
 }
+
+export const [innerReducer, outerReducer] = [true, false].map(pathReducer)
+export const getter = from => route =>
+  createReducer([always(toArray(route)), from], path)
```
Comment (voliva) [inline src/path.js:35]: This name (`getter`) is a bit unclear of what this does.... I've thought that maybe we could change this for "createPath = from => route => path" or something similar

Comment (josepot) [reply src/path.js:35]: I agree that `getter` is definitely not the best name. I was a bit lazy when I created it :see_no_evil: In my defense, at least it is not being exported :sweat_smile: 

But yeah, I agree. We should find a better name. Maybe we should even consider exporting it. After giving it some thought, what about `getViewerFor`? That name is slightly inspired [by the lens of ramda](https://ramdajs.com/docs/#lens).

Comment (voliva) [reply src/path.js:35]: Ah yes, that's a good name :+1:

**Review by voliva — APPROVED** [review]

**Pull request merged**